pkg/network/protocols/http2/protocol.go

```diff
@@ -343,6 +353,47 @@ func (p *protocol) GetStats() *protocols.ProtocolStats {
 	}
 }
 
+// UpdateKernelTelemetry should be moved to the HTTP/2 part as well
+func (p *protocol) UpdateKernelTelemetry(mgr *manager.Manager) {
```

Comment:
why is the function exported?

Reply:
changed, I used it in a test and forgot to return it